run_matching.R

```diff
@@ -113,11 +115,14 @@ if (!file.exists(path_abcd)) {
   stop(glue::glue("No ABCD file found at path {path_abcd}. Please check your project setup!"))
 }
 
-abcd <- readr::read_csv(
-  file.path(path_abcd),
-  col_types = col_types_abcd,
-  col_select = dplyr::all_of(col_select_abcd)
+abcd <- readxl::read_xlsx(
+  path = file.path(path_abcd),
+  sheet = sheet_abcd,
+  col_types = cols_abcd$col_types_abcd
 )
+if (!all(cols_abcd$col_names_abcd %in% names(abcd))) {
+  stop("Columns in abcd do not match expected input names. Please check your input.")
+}
```

Comment:
Shouldn't you use the "prepared" abcd for matching? 

If we are  potentially removing inactive companies anyway, it could lead to a slightly easier matching process.

Reply:
But if not, then this LGTM :-)

Reply:
This is something I have been wondering for a while. On the other hand, it is not necessary to remove inactive companies from standard PACTA (it becomes a necessity for the aggregate metrics) and removing them from the start would potentially reduce the match success rate. Which may be fair, but it is not obvious to me. I think we can do that, which may indicate that both prep_* scripts in the mural should move out of the main.R script

Reply:
Hmm yeah that's fair. 
We can leave it as-is for now and discuss in a different forum if you like?